frameworks/Java/officefloor/src/woof_benchmark/src/main/java/net/officefloor/benchmark/Logic.java

```diff
@@ -36,76 +32,29 @@ public void db(EntityManager entityManager, ObjectResponse<World> response) {
 
 	// ========== QUERIES ==================
 
-	public void queries(@HttpQueryParameter("queries") String queries, QueriesFlows flows,
+	public void queries(@HttpQueryParameter("queries") String queries, EntityManager entityManager,
 			ObjectResponse<World[]> response) {
 		ThreadLocalRandom random = ThreadLocalRandom.current();
-		int[] loaded = new int[] { 0 };
 		int count = getQueryCount(queries);
 		World[] worlds = new World[count];
 		for (int i = 0; i < worlds.length; i++) {
-			int index = i;
-			GetEntry entry = new GetEntry(random.nextInt(1, 10000));
-			flows.getEntry(entry, (escalation) -> {
-				worlds[index] = entry.world;
-				loaded[0]++;
-				if (loaded[0] >= count) {
-					response.send(worlds);
-				}
-			});
+			worlds[i] = entityManager.find(World.class, random.nextInt(1, 10000));
```

Comment:
Sorry we missed this before, but ThreadLocalRandom's `nextInt()` upper bound is exclusive. Could you change all these to 10001

Reply:
No worries... will update PR :+1: